keyboards/monarch/config.h

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2015 Jun Wako <wakojun@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4011
+#define PRODUCT_ID      0x43C1
+#define DEVICE_VER      0x0001
+#define MANUFACTURER DoCallMeKing
+#define PRODUCT Monarch
+
+/* key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 16
+
+#define MATRIX_COL_PINS { A10, A9, A8, B15, B14, B13, B12, B10, B2, B1, B0, A5, A7, A4, A3, B6 }
+#define MATRIX_ROW_PINS { A15, B3, B11, A2, A1, B9 }
+#define DIODE_DIRECTION COL2ROW
+
+/* Rotary encoder pins */
+#define ENCODERS_PAD_A { B5 }
+#define ENCODERS_PAD_B { B4 }
+#define ENCODER_RESOLUTION 1
+#define ENCODER_DIRECTION_FLIP
```

Comment:
```suggestion
#define ENCODERS_PAD_A { B4 }
#define ENCODERS_PAD_B { B5 }
#define ENCODER_RESOLUTION 1
```

Reply:
Am I good to go now? I think I’ve updated all necessary files if you get latest.